lib/burndown_chart.rb

```diff
@@ -114,9 +114,10 @@ def setup(burndown_dir, board_id)
     write_data File.join(burndown_dir, burndown_data_filename)
   end
 
-  def self.plot(sprint_number, working_dir=".")
+  def self.plot(sprint_number, options)
     plot_helper = File.expand_path("../../scripts/create_burndown.py", __FILE__ )
-    system "python #{plot_helper} #{sprint_number} #{working_dir}"
+    cli_switches = options.keys.map{|k| k.dup.prepend('--')}
```

Comment:
I would prefer to make this explicit and check the options, which are passed on. It might also be good to add a test for this method.

With this change it doesn't work, because `trollolo` accepts an `--output` option, but the python rendering script expects a `--working-dir`.